rpc/src/rpc_actor.rs

```diff
@@ -40,6 +41,7 @@ pub struct RpcCollectedState {
 pub struct RpcServer {
     shell_channel: ShellChannelRef,
```

Comment:
@mambisi 
I guess, we dont need shell_channel here, because it is in env, please remove it

Reply:
Ok

Reply:
We need it

https://github.com/mambisi/tezedge/blob/c440246edbfca0e93954c04ad6a58597d082ad15/rpc/src/server/shell_handler.rs#L297

Reply:
@mambisi 
yes, we need it, but it is used from `env.shell`,
I want to just remove it from `pub struct RpcServer {` and where need it, just it it from `env`